airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -136,7 +141,16 @@ public void start() throws IOException {
         SCHEDULING_DELAY.toSeconds(),
         TimeUnit.SECONDS);
 
-    scheduledPool.scheduleWithFixedDelay(
+    executeJobsPool.scheduleWithFixedDelay(
```

Comment:
fyi @jrhizor @cgardens splitting all of these into separate thread pools as per the result of my stress test